.github/workflows/manual.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy
+
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
Can we add predefined inputs to offer a nice dropdown menu?
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs

Reply:
Using input type as environment creates the dropdown with the environments available on the repo without specifying input options. Added input options now to make clear which environments are available.